src/lava/lib/optimization/solvers/generic/solution_finder/models.py

```diff
@@ -77,11 +77,18 @@ def __init__(self, proc):
         proc.vars.variables_assignment.alias(
             self.variables.variables_assignment
         )
-        proc.vars.cost.alias(
-            self.cost_convergence_check.cost
```

Comment:
if the name is explicit about last and first being bits or bytes, you would not need the comment